repose-aggregator/components/filters/herp/src/test/scala/org/openrepose/filters/herp/HerpFilterTest.scala

```diff
@@ -154,6 +213,19 @@ class HerpFilterTest extends FunSpec with BeforeAndAfterAll with BeforeAndAfter
       logEvents.size shouldBe 1
       logEvents.get(0).getMessage.getFormattedMessage should include("\"Method\" : \"POST\"")
     }
+    it("should expose the cadf method") {
+      // given:
+      servletRequest.setMethod("POST")
+
+      // when:
+      herpFilter.configurationUpdated(herpConfig)
+      herpFilter.doFilter(servletRequest, servletResponse, filterChain)
+
+      // then:
+      def logEvents = listAppenderPre.getEvents
+      logEvents.size shouldBe 1
+      logEvents.get(0).getMessage.getFormattedMessage should include("\"CadfMethod\" : \"update/post\"")
+    }
```

Comment:
Should this be in foreach loop to test all of the conditions and increase code coverage?